### PR TITLE
update readme to add installation by asdf

### PR DESCRIPTION
**Pull request by koluku**
I created asdf plugin as alternative installation method.
It works different version binary installation and execution like `v1` and `v2`.

https://github.com/koluku/asdf-ecspresso

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -442,7 +450,7 @@ $ ecspresso --config ecspresso.yml diff --unified
 -  "platformVersion": "1.3.0"
 +  "platformVersion": "LATEST"
  }
-
```
Comment (fujiwara) [inline README.md]: @koluku revert this change (white space is deleted) please.

Comment (koluku) [reply README.md]: oh, sorry. I reverted it.

**Review by fujiwara — APPROVED** [review]
